Added accordion collapse using bootstrap

mostlybadfly opened the pull request:
Trying this again.  Incorporated bootstrap to work with each `missing_count`

tomekr commented:
It looks like as it stands now, this is actually creating a new panel-group for each missing ingredient list. The panel-group div is meant to contain all of the missing ingredient groups (i.e. 1 missing ingredient, 2 missing ingredients, etc.). That's why you there is spacing between each group instead of having it all in the same accordion. Also, the CSS here seems to be a bit wonky, because it should look and function exactly as the [example](http://getbootstrap.com/javascript/#collapse) looks. I'll see if I can figure out what's up with that.

mostlybadfly commented:
I know that I hardcoded the missing_count into the tags for this, I wonder if there not being a certain amount of missing ingredients is impacting this?  Could also be a tag that is not encapsulating them. Just some thoughts as I go back to this.

mostlybadfly commented:
I moved the `panel-group` further up outside of the loop, but now it is surrounding "Drinks you can make", It probably doesn't go there, but the accordion is spaced better.  pushing for now and will play with it more.  Is there a way to escape the loop and make that a fixed html element, does that make sense? I could also create two separate loops and do something like `if missing_count > 0`?

tomekr commented on app/views/drinks/makeable_drinks.html.erb:
```diff
@@ -19,25 +20,38 @@
+            <div class="panel-heading" role="tab" id="heading<%= missing_count.to_s %>">
+              <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse<%= missing_count.to_s %>" aria-expanded="false" aria-controls="collapse<%= missing_count.to_s %>">
+                  <h4>Missing <%= missing_count %> <%= "Ingredient".pluralize(missing_count) %></h4>
```
Possible extra <h4> element here. Check out http://getbootstrap.com/javascript/#collapse

mostlybadfly commented:
I updated the User model to allow a specification of missing_count and have updated the controller.  The accordion was pulling the main up and the body color was being revealed so I changed the body background color to match.  I'm assuming this is ok but let me know if not.

tomekr commented:
Looks good! :+1: I'll work on merging this later unless you want to take a stab at merging.

mostlybadfly commented:
Great! I will take a look.

mostlybadfly commented:
Really hope I merged that ok, please take a look :)

tomekr commented:
Yaay, merge looks good to me. Thanks again! :+1:
